samples/samples/snippets.py

```diff
@@ -1407,6 +1407,29 @@ def insert_singers(transaction):
 # [END spanner_get_commit_stats]
 
 
+# [START spanner_set_max_commit_delay]
```

Comment:
can you add a test for sample to validate its working similar to below?
https://github.com/googleapis/python-spanner/blob/d5acc263d86fcbde7d5f972930255119e2f60e76/samples/samples/snippets_test.py#L494

Reply:
Done.